plugin/Renderer/InputFieldCheckboxRenderer.php

```diff
@@ -124,7 +127,8 @@ public function render()
 				.'" value="'.esc_html($this->getValue()).'"'
 				.($this->getValue() == $this->_checkedValues ? ' checked="checked" ' : '')
 				.$this->renderAdditionalAttributes()
-				.' id="'.esc_html($this->getGuiId()).'">';
+				.' id="'.esc_html($this->getGuiId()).'">'
+				.(!empty($this->_descriptionTextHTML) && is_string($this->_descriptionTextHTML) ? '<p class="description">'.$this->_descriptionTextHTML.'</p><br>' : '');
```

Comment:
better check for `!== ''` instead of `is_string()`

Reply:
ok I will update